spec/system/admin/reports/orders_and_distributors_spec.rb

```diff
@@ -0,0 +1,188 @@
+# frozen_string_literal: true
+
+require "system_helper"
+
+RSpec.describe "Orders And Distributors" do
+  include AuthenticationHelper
+  include WebHelper
+  include ReportsHelper
+
+  describe "Orders And Distributors" do
+    let!(:report_url) { admin_report_path(report_type: :orders_and_distributors) }
+    let!(:distributor) { create(:distributor_enterprise, name: "By Bike") }
+    let!(:distributor2) { create(:distributor_enterprise, name: "By Moto") }
+    let!(:completed_at) { Time.zone.now.to_fs(:db) }
+
+    around do |example|
+      Timecop.travel(completed_at) { example.run }
+    end
```

Comment:
This is not freezing time. It's just going to the beginning of the second, I guess. Therefore flakiness is less likely but not impossible.